integration-tests/contract-edo-deploy-having-ticket.spec.ts

```diff
@@ -1,28 +1,71 @@
 import { CONFIGS } from "./config";
 import { ticketCode, ticketStorage } from './data/code_with_ticket';
+import { ticketCodeProto14, ticketStorageProto14 } from './data/code_with_ticket_proto14';
+import { Protocols } from "@taquito/taquito";
 
```

Comment:
could we change the file name here to not include the `edo` protocol name? perhaps something like`contract-deploy-with-ticket.spec.ts`

Reply:
pushed a commit to update the file name